scripts/Packages/Package_Gullwing__QFP_SOIC_SO/size_definitions/sot.yaml

```diff
@@ -0,0 +1,15 @@
+FileHeader:
+  library_Suffix: 'TO_SOT_SMD'
+  device_type: 'SOT'
+
+SOT-23-6:
+  size_source: 'https://www.jedec.org/sites/default/files/docs/Mo-178c.PDF variant AB'
+  custom_name_format: 'SOT-23-6'
+  body_size_x: 1.6 +/-0.15
+  body_size_y: 2.9 +/-0.15
+  overall_size_x: 2.8 +/-0.20
```

Comment:
Please remove tolerances, these are given as basic dimensions.
aaa and bbb are surface tolerances, not length.